cassiopeia/data/CassiopeiaTree.py

```diff
@@ -348,7 +348,7 @@ def root(self) -> str:
 
         if "root" not in self.__cache:
             self.__cache["root"] = [
-                n for n in self.__network if self.in_degree(n) == 0
+                n for n in self.__network if self.__network.in_degree(n) == 0
```

Comment:
For consistency, why don't we just use `[n for n in self.__network if self.is_root(n)]`?